{{cookiecutter.repo_name}}/openedx.yaml

```diff
@@ -0,0 +1,12 @@
+# openedx.yaml
+
+---
+owner: jmbowman
```

Comment:
Can this be parameterized? Otherwise you might end up accidentally being the owner of all new repos.